### PR TITLE
Add /up webhook, make it more dynamic so you can have more services

### PR DESCRIPTION
**Pull request by Dinoshauer**

### CODE REVIEW

**Comment by joshuakarjala** [comment]
add dependencies to sealand.js

**Comment by Dinoshauer** [comment]
done

File: src/sealand.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 //   Allow you to do Rancher deploys from your favourite chat CLI
 //
 // Dependencies:
-//   docker-registry-client": "^3.0.1"
+//  "async-each": "^1.0.0"
```
Comment (joshuakarjala) [inline src/sealand.js]: whitespace

Comment (Dinoshauer) [inline src/sealand.js]: fixed

**Pull request merged**